.eslintrc.js

```diff
@@ -187,6 +188,7 @@ module.exports = {
       rules: {
         '@typescript-eslint/no-unsafe-call': 'off',
         '@typescript-eslint/no-unsafe-return': 'off',
+        '@typescript-eslint/restrict-plus-operands': 'off',
```

Comment:
Why do we need to disable this rule?

Reply:
With this rule enabled, I get the following error:
```
[...]/browser-sdk/eslint-local-rules/disallowProtectedDirectoryImport.js
  85:64  error  Operands of '+' operation must either be both strings or both numbers  @typescript-eslint/restrict-plus-operands
```
This is because TS tries to infer types in JS files, but often it fails to do so and just uses `any` in most places. This rule forbids to use the `+` operator with `any` types. Same thing as what I tried to document [above](https://github.com/DataDog/browser-sdk/pull/1235/files/43449ab679af405fd24d124d36f11ffbec647ca2#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R186),

Reply:
Got it ;)